src/components/DashboardLayout.tsx

```diff
@@ -12,6 +13,12 @@ const Layout: FunctionComponent = ({ children }) => {
 
   return (
     <>
+      <a
+        href="#main-content"
```

Comment:
Does this page need an element with `#main-content` or does it exist already?

Reply:
it's the main on line 58